ngraph/core/include/ngraph/op/range.hpp

```diff
@@ -23,6 +23,43 @@ namespace ngraph
 {
     namespace op
     {
+        namespace v4
+        {
+            /// \brief Range operation, analogous to `arange()` in Numpy.
+            class NGRAPH_API Range : public Op
+            {
+            public:
+                static constexpr NodeTypeInfo type_info{"Range", 4};
```

Comment:
Please use NGRAPH_RTTI_DECLARATION and  NGRAPH_RTTI_DEFINITION
In hpp file: NGRAPH_RTTI_DECLARATION;
In cpp file: NGRAPH_RTTI_DEFINITION(op::v4::Range, "Range", 4);